platform/broadcom/sonic-platform-modules-inventec/common/utils/transceiver_monitor.py

```diff
@@ -229,77 +264,115 @@ def set_power_mode_for_QSFP(self):
         for index in self.get_port_to_i2c_mapping().keys():
             if index >= self.qsfp_port_start and index <= self.qsfp_port_end :
                 self.set_low_power_mode(index, False)
+            else:
+                # To set tx_disable
+                self.set_tx_disable(index)
```

Comment:
>set_tx_disable [](start = 21, length = 14)

why do you set tx_disable here?

Reply:
Sorry for the mistake. For thei ndex outside the range (port_start, port_end), they are SFP ports. set_tx_disable is to write 0 to TX_DISABLE of SFP transceiver while set_low_power_mode is for turning off Low power mode of QSFP transceiver.